src/Exports/Excel.php

```diff
@@ -204,7 +204,7 @@ private function addNewSheet(): void
 
     private function needsNewSheet(): bool
     {
-        return $this->dataExport->entries / ConfigFacade::get('enso.tables.export.sheetLimit')
+        return optional($this->dataExport)->entries / ConfigFacade::get('enso.tables.export.sheetLimit')
```

Comment:
here we're losing the sheetLimit.

We should use an internal entry count when dataexport is not available